builder/go/main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import "fmt"
+
+func main() {
+	LessonBuilder := NewNewStudentLessonBuilder()
+	LessonDirector := NewLessonDirector(LessonBuilder)
+	LessonDirector.make()
```

Comment:
Make and getResult methods should be exported because these are used by main package 🙏🏻